sleuth_crawler/scraper/scraper/spiders/parsers/utils.py

```diff
@@ -0,0 +1,30 @@
+from bs4 import BeautifulSoup
+
+def strip_content(data):
+    """
+    Remove garbage from raw web page content, return as list of lines
+    """
+    try:
+        # strip JavaScript, HTML
+        soup = BeautifulSoup(data, "html.parser")
+        for script in soup(["script", "style"]):
+            script.decompose()
+        data = soup.get_text()
+        # strip extraneous line breaks and sort into list
+        lines = []
+        for line in data.splitlines():
+            line = line.strip()
+            if line:
+                lines.append(line.strip())
+        return lines
+    except Exception:
+        return None
```

Comment:
Put a comment here mentioning any it's ok to abort when there are exceptions. Either that or handle exceptions case-by-case.

Reply:
fixed in 779ea0c